spec/mongoid/tasks/models.rb

```diff
@@ -0,0 +1,13 @@
+module DatabaseSpec
```

Comment:
maybe name this file `database_spec_models.rb` for consistency with the rest of the codebase?

Reply:
Good point, thanks. Updated.